domains/onichan.json

```diff
@@ -0,0 +1,11 @@
+{
+  "description": "onichan's plugins. what can i say?",
+  "repo": "https://github.com/rbstsoul",
+  "owner": {
+    "username": "rbstsoul",
+    "email": "rbstsoul@gmail.com"
+  },
+  "record": {
+    "URL": "rbst.gitbook.io/plugin"
```

Comment:
url must be a valid url (starts with http:// or https://

Reply:
I'm fixing it